src/templates/index.pug

```diff
@@ -8,6 +14,9 @@ body.layout
     // Page header
     nav.layout__header#header
         a.layout__header__menu(class=['logo'])="C++ Tour"
+        div.layout__header__icons
```

Comment:
For now class here server no purpose so you can remove it.
I'm talking about `layout__header__icons`

Reply:
I didn't have it in the first place! You made me add it. :stuck_out_tongue:

Reply:
Sorry about that I was thinking either add it or class for class :)
But since we settled for icon class then we remove it :)